docs/obs/traffic-inspection.mdx

```diff
@@ -48,6 +48,12 @@ Once Full Capture is enabled, a replay button will appear for fully captured req
 
 Replay works by resending your original unmodified request to your endpoint. It will be subject to the policies of that endpoint at the time of replay (which could be different than the original request).
 
+:::warning
+
+You need to use a permanent endpoint for your service for replay to work. If you are using a free ngrok account, where the URL changes every time you restart the agent, the replay request will fail as it tries to contact an endpoint that no longer exists.
```

Comment:
Isn't this only true if you've stopped or restarted your agent since the request was logged in Traffic Inspector?